data/binding/gen.go

```diff
@@ -388,10 +388,17 @@ type {{ .Name }}List interface {
 // External{{ .Name }}List supports binding a list of {{ .Type }} values from an external variable.
 //
 // Since: {{ .Since }}
+{{- if eq .Name "Untyped" }}
+type UntypedItemComparator func(a interface{}, b interface{}) bool
```

Comment:
This type definition could (should?) be outside of the generated files, as it is not created based on each type being generated